### DISCUSSION TITLE
unable to link this library to my vs code

### DISCUSSION THREAD

**Discussion by shreya-spr** [AndreRenaud/PDFGen#129] [Q&A] [answered]
Have downloaded this library and copied it in the mingw64/include path. But vsc does not seem to recognize the 
`#include<pdfgen.h>`  in my  c file project . Could you help me with this? And I have Windows OS

**Comment by AndreRenaud** [comment]
Do you have the error message that it produces? If you've put pdfgen.c & pdfgen.h in the same folder as the code where you're trying to use it, you should probably use quotes for the #include, not `<`
ie: `#include "pdfgen.h"` instead of `#include <pdfgen.h>`

**Reply by shreya-spr** [reply]
Yes, pdfgen.c and pdfgen.h are in the same folder as my project. I used `#include "pdfgen.h"` I compiled it: gcc <filename.c>
And the error message I get is this
<img width="682" alt="image" src="https://user-images.githubusercontent.com/92443502/173355250-576ddad7-a071-4d22-a4f1-2a86d816f71c.png">

**Comment by AndreRenaud** [comment, marked as answer]
It looks like you're not linking pdfgen.c with your other C files (billing.c by the looks of it). That output doesn't show the actual command line that you used, but I'm assuming it's something like `gcc billing.c`. That isn't enough - you either need to compile each .c file into it's own .o file separately, ie: `gcc -o billing.o billing.c`, `gcc -o pdfgen.o pdfgen.c` and then link them using `gcc -o myprogram billing.o pdfgen.o`, or you need to compile/link them all in one step, ie: `gcc -o myprogram billing.c pdfgen.c`. Are you using a `Makefile` or some other build tool?
